fmriprep/cli/parser.py

```diff
@@ -881,10 +881,9 @@ def parse_args(args=None, namespace=None):
     if output_dir == bids_dir:
         parser.error(
             'The selected output folder is the same as the input BIDS folder. '
-            'Please modify the output path (suggestion: %s).'
-            % bids_dir
+            f'Please modify the output path (suggestion: {bids_dir}).'
             / 'derivatives'
-            / ('fmriprep-%s' % version.split('+')[0])
+            / ('fmriprep-{}'.format(version.split('+')[0]))
```

Comment:
```suggestion
```

Reply:
Not sure about this one. >You don't want to delete this line, do you?

Reply:
Yes, I bundled it into the previous suggestion.